src/styles/blocks/card/blockscard/cardBuy.scss

```diff
@@ -0,0 +1,24 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    box-sizing: border-box;
```

Comment:
There's no need to repeat the `box-sizing: border-box;` property in the `:hover` state, as it is already declared in the base class. This redundancy can make your code harder to maintain.